28.05.2019 10:20:20 Checking for system tray...
28.05.2019 10:20:20 Starting pgAdmin4 server...
28.05.2019 10:20:20 Creating server object, port:53226, key:500c071f-b646-454c-8ea0-7009e3670cb1, logfile:C:/Users/alxarsnk/.pgAdmin4.2123502176.log
28.05.2019 10:20:20 Python Path: C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/Lib/site-packages;C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/DLLs;C:/Program Files/PostgreSQL/10/pgAdmin 4/venv/Lib
28.05.2019 10:20:20 Python Home: C:/Program Files/PostgreSQL/10/pgAdmin 4/venv
28.05.2019 10:20:20 Initializing Python...
28.05.2019 10:20:20 Python initialized.
28.05.2019 10:20:20 Adding new additional path elements
28.05.2019 10:20:20 Redirecting stderr...
28.05.2019 10:20:20 stderr redirected successfully.
28.05.2019 10:20:20 Initializing server...
28.05.2019 10:20:20 Webapp Path: C:/Program Files/PostgreSQL/10/pgAdmin 4/web/pgAdmin4.py
28.05.2019 10:20:20 Server initialized.
28.05.2019 10:20:20 Starting Server Thread...
28.05.2019 10:20:20 Open the application code and run it.
28.05.2019 10:20:20 Set the port number, key and force SERVER_MODE off
28.05.2019 10:20:20 PyRun_SimpleFile launching application server...
28.05.2019 10:20:21 Application Server URL: http://127.0.0.1:53226/?key=500c071f-b646-454c-8ea0-7009e3670cb1
28.05.2019 10:20:21 The server should be up, we'll attempt to connect and get a response. Ping the server
28.05.2019 10:20:50 Attempt to connect one more time in case of a long network timeout while looping
28.05.2019 10:20:50 Everything works fine, successfully started pgAdmin4.
